<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TNNewFriendCard</class>
 <widget class="QWidget" name="TNNewFriendCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>248</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>652</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="finishedMarkLabel">
       <property name="maximumSize">
        <size>
         <width>58</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>已完成</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="mainWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <widget class="QLabel" name="timeLabel">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>15</height>
               </size>
              </property>
              <property name="text">
               <string>日期时间</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="subCatalogLabel">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>15</height>
               </size>
              </property>
              <property name="text">
               <string>subCatalog</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="infoWidget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1,0">
               <item>
                <widget class="QWidget" name="avatarWidget" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="topMargin">
                   <number>1</number>
                  </property>
                  <property name="bottomMargin">
                   <number>1</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <property name="leftMargin">
                     <number>10</number>
                    </property>
                    <property name="topMargin">
                     <number>5</number>
                    </property>
                    <property name="rightMargin">
                     <number>10</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_2">
                      <item>
                       <widget class="TNAvatorButton" name="herAvatarButton">
                        <property name="minimumSize">
                         <size>
                          <width>40</width>
                          <height>40</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>40</width>
                          <height>40</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>对方</string>
                        </property>
                        <property name="flat">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout">
                        <property name="spacing">
                         <number>0</number>
                        </property>
                        <property name="topMargin">
                         <number>4</number>
                        </property>
                        <property name="bottomMargin">
                         <number>4</number>
                        </property>
                        <item>
                         <widget class="TNElidedLabel" name="herNickNameLabel">
                          <property name="text">
                           <string>我的小鱼你醒了</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="TNElidedLabel" name="herSubTitleLabel">
                          <property name="text">
                           <string>快乐生活，快乐学习。</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_3">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="TNAvatorButton" name="mineAvatarButton">
                      <property name="minimumSize">
                       <size>
                        <width>40</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>40</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>我方</string>
                      </property>
                      <property name="flat">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="textWidget" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>60</height>
                  </size>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <property name="topMargin">
                   <number>1</number>
                  </property>
                  <property name="bottomMargin">
                   <number>1</number>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_2">
                    <property name="leftMargin">
                     <number>20</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="summeryLabel">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>summery</string>
                      </property>
                      <property name="indent">
                       <number>0</number>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="textLabel">
                      <property name="text">
                       <string>text</string>
                      </property>
                      <property name="wordWrap">
                       <bool>true</bool>
                      </property>
                      <property name="indent">
                       <number>0</number>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="TNElidedLabel" name="resourceLabel">
                      <property name="text">
                       <string>resource</string>
                      </property>
                      <property name="indent">
                       <number>0</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="buttonWidget" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>3</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QPushButton" name="acceptButton">
                      <property name="text">
                       <string>接受</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="rejectButton">
                      <property name="text">
                       <string>拒绝</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="buttonNameButton">
                      <property name="text">
                       <string>buttonName</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TNElidedLabel</class>
   <extends>QLabel</extends>
   <header location="global">TNElidedLabel.h</header>
  </customwidget>
  <customwidget>
   <class>TNAvatorButton</class>
   <extends>QPushButton</extends>
   <header location="global">tnavatorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
